src/clique_parser.erl

```diff
@@ -380,7 +380,7 @@ parse_valid_arg_value_with_equal_sign_test() ->
 %% All arguments must be of type k=v
 parse_invalid_kv_arg_test() ->
     Spec = spec(),
-    Args = ["ayo"],
+    Args = ["ayo="],
```

Comment:
Can we test both cases?

Reply:
Thanks for the quick response, I'll change the test to support both cases as soon as I get time.

Reply:
Done